pkg/bootstrapper/bootstrapper.go

```diff
@@ -503,29 +526,41 @@ func (wmcb *winNodeBootstrapper) createKubeletService() error {
 	return nil
 }
 
-// startKubeletService starts the kubelet as a Windows service
-func (wmcb *winNodeBootstrapper) startKubeletService() error {
-	if wmcb.kubeletSVC == nil {
-		return fmt.Errorf("no kubelet service")
+// startService starts a Windows service
+func (wmcb *winNodeBootstrapper) startService() error {
```

Comment:
This function confuses me. It says it `starts a Windows service`, but doesn't give input for which service.
I feel like this should be a method of the `service` object. It can then be renamed to `start()`